--- dbConfig_a.php.original	2014-09-06 13:06:20.000000000 -0700
+++ dbConfig_a.php	2014-09-06 13:08:24.000000000 -0700
@@ -219,7 +219,6 @@
     <td>&nbsp;</td>
     <td nowrap><b>{$mod_strings['LBL_DBCONF_DB_PASSWORD']}</b></td>
     <td nowrap align="left"><input type="password" name="setup_db_sugarsales_password_entry" value="{$setup_db_sugarsales_password}" /><input type="hidden" name="setup_db_sugarsales_password" value="{$setup_db_sugarsales_password}" /></td>
-    <input type="hidden" name="setup_db_sugarsales_password" value="{$_SESSION['setup_db_sugarsales_password']}" /></td>
 </tr>
 <tr>
     <td>&nbsp;</td>
@@ -489,4 +488,4 @@
     echo $out_dd;
 echo $out5;

-?>
\ No newline at end of file
+?>
